exercism/ocr-numbers/ocr-number.js

```diff
@@ -0,0 +1,125 @@
+var Ocr = function(){}
+
+  Ocr.prototype.convert = function(ocrString) {
+    var ocrSplit = ocrString.split('\n');
+
+    var choppedOcr = [];
+    var ocrLoop = (function(){
+      for(i=0; i < ocrSplit.length; i++ ){
+        var nestedOcr = ocrSplit[i].match(/.{1,3}/g);
+        choppedOcr.push(nestedOcr);
+      }
+      return choppedOcr;
+    });
+    ocrLoop();
+
+    var finalOcr = [];
+    var ocrFinal = (function(){
+      for(var x=0; x < choppedOcr[0].length; x++){
+         for(var i=0; i < choppedOcr.length; i++){
+           finalOcr.push(choppedOcr[i][x]);
+        }
+      }
+      return finalOcr;
+    });
+    ocrFinal();
+
+
+    var convertedOcr = [];
+    var ocrMachine = function(){
+      while(finalOcr.length > 0){
+        var ocrNumber = finalOcr.slice(0,4).join('\n');
+          if(ocrNumber ==
```

Comment:
maybe try switch statements next time? This might be too many `else if` statements.